src/main/java/com/sforce/ws/ConnectorConfig.java

```diff
@@ -161,9 +168,23 @@ public void close() throws IOException {
     private String ntlmDomain;
 	private TransportFactory transportFactory;
 	  private SSLContext sslContext;
-
+    private final static Version VERSION_14_0_0_0 =
+            Version.Builder.newBuilder().setFeature(14).build();
+    private final static Version VERSION_13_0_2 =
+            Version.Builder.newBuilder().setFeature(13).setInterim(0).setUpdate(2).build();
+    private final static Version VERSION_11_0_6 =
+            Version.Builder.newBuilder().setFeature(11).setInterim(0).setUpdate(6).build();
+    private final static Version VERSION_1_8_0_0_321 =
+            Version.Builder.newBuilder().setFeature(1).setInterim(8).setUpdate(0).setPatch(0).setBuild(321).build();
+    private static boolean javaVersionHasBug = doesVersionHasABug(System.getProperty(JAVA_VERSION_PROPERTY));
```

Comment:
I think it would be good to overload this method:
```
boolean doesVersionHasABug() {
    return doesVersionHasABug(System.getProperty(JAVA_VERSION_PROPERTY));
}
```